lib/core/test/unit/Cardano/Wallet/Primitive/AddressDiscovery/RandomSpec.hs

```diff
@@ -188,16 +189,16 @@ goldenSpecTestnet =
 -------------------------------------------------------------------------------}
 
 data GoldenTest = GoldenTest
-    { mnem :: [Text]
+    { mnem :: SomeMnemonic
     , addr :: ByteString
     , accIndex :: Word32
     , addrIndex :: Word32
     , expected :: Bool
     } deriving (Show, Eq)
 
 -- An arbitrary mnemonic sentence for the tests
-arbitraryMnemonic :: [Text]
-arbitraryMnemonic =
+arbitraryMnemonic :: SomeMnemonic
+arbitraryMnemonic = either (error . show) id $ fromMnemonic @'[12]
```

Comment:
Having to explicitly tell (`@'[12]`) which mnemonic lengths are supported, when we are hard-coding the mnemonic, could be considered cumbersome.

Reply:
Alternatively, use: `unsafeMkMnemonic` from `Cardano.Wallet.Unsafe`